Unmasked/Unmasked View/View Controllers/UnmaskedViewController.swift

```diff
@@ -0,0 +1,259 @@
+//
+//  UnmaskedViewController.swift
+//  Unmasked
+//
+//  Created by Junaid Rajah on 2021/11/19.
+//
+
+import UIKit
+import UnmaskedEngine
+
+class UnmaskedViewController: UIViewController {
+    
+    private var unmaskedViewModel: UnmaskedViewModel?
```

Comment:
Once you've made the change I have suggested below, you can make this `private lazy` will also fix the force unwrapping.